fusor-ember-cli/app/adapters/deployment.js

```diff
@@ -1,5 +1,12 @@
 import ApplicationAdapter from './application';
 
 export default ApplicationAdapter.extend({
-    namespace: 'fusor/api/v21'
+  namespace: 'fusor/api/v21',
+
+  urlForQueryRecord(query, modelName) {
+    if (query.deployment_id) {
+      return `/${this.get('namespace')}/deployments/${query.deployment_id}`;
+    }
```

Comment:
@isratrade so I believe this is the deployment adapter since it's in deployment.js.  I was trying to find a way to send a get to a single resource with parameters and Ember didn't seem to give that capability.

/deployments/1?sync=true (show with params)
not
/deployments?sync=true (index with params from queryRecord)
or
/deployments/1  (show no params from findRecord)

I should have however been using query.id, not query.deployment_id.  So I'm changing that.

Also, if we want to make it clearer from the code we could change our adapters to be:

```javascript
const DeploymentAdapter = ApplicationAdapter.extend({
...
});

export default DeploymentAdapter;
```